rclpy/test/test_node.py

```diff
@@ -75,15 +75,15 @@ def raw_subscription_callback(self, msg):
     def test_create_raw_subscription(self):
         executor = SingleThreadedExecutor(context=self.context)
         executor.add_node(self.node)
-        primitives_pub = self.node.create_publisher(Primitives, 'raw_subscription_test')
+        primitives_pub = self.node.create_publisher(BasicTypes, 'raw_subscription_test')
```

Comment:
Maybe change the name of the variables too?

Same below: `primitives_msg`.

Reply:
Missed it! Changed be3848c